microsoft/testsuites/core/disk_demo.py

```diff
@@ -0,0 +1,69 @@
+from typing import Any
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+    search_space,
+)
+from lisa.features import Disk
+from lisa.features.disks import DiskStandardHDDLRS
```

Comment:
Can you please run `make check` and fix errors? Also, please enable formatting with black

Reply:
I had not setup my environment properly. It is done so now and appears to be fixed.

Reply:
Ran `make check` and enabled formatting all issues were resolved